stdlib/ballerina-http/src/main/ballerina/ballerina/net/ws/natives.bal

```diff
@@ -47,6 +47,11 @@ public struct HandshakeConnection {
     map<string> upgradeHeaders;
 }
 
+@Description { value:"Gets the query parameters from the HandshakeConnection as a map"}
+@Param { value:"req: The HandshakeConnection struct" }
+@Return { value:"The map of query params" }
+public native function <HandshakeConnection conn> getQueryParams () (map);
```

Comment:
Formatting issue - space after {

Reply:
Is there a convention on this? I see lots of spaces in this file also: https://github.com/ballerina-lang/ballerina/blob/master/stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

Reply:
Convention we followed is similar to what we get if we format the file using intellij idea plugin. You can try with that and please fix if there are any other places in the same file which does not follow the format.